config/opal_check_cuda.m4

```diff
@@ -90,7 +90,7 @@ AS_IF([test "$opal_check_cuda_happy" = "yes"],
 
 # If we have CUDA support, check to see if we have support for SYNC_MEMOPS
 # which was first introduced in CUDA 6.0.
-AS_IF([test "$opal_check_cuda_happy"="yes"],
+AS_IF([test "$opal_check_cuda_happy" = "yes"],
```

Comment:
The syntax error check PR appears to already be in main.  Did you mis-rebase somewhere?

Reply:
Let me check